docs/docs-beta/docs/guides/deployment/kubernetes.md

```diff
@@ -1,4 +1,215 @@
+
+:::note
+If you are running an older version of Dagster, pass the --version flag to `helm upgrade` with the version of Dagster you are running. For example, if you are running `dagster==1.7.4` you'll run the command `helm upgrade --install dagster dagster/dagster -f /path/to/values.yaml --version 1.7.4`
+:::
+
+The `helm upgrade` command will launch several pods in your Kubernetes cluster. You can check the status of the pod with the command:
+
+```bash
+kubectl get pods
+```
+
+It may take a few minutes before all pods are in a `RUNNING` state. If the `helm upgrade` was successful, you should see a `kubectl get pods` output similar to this:
+
+```bash
+$ kubectl get pods
+NAME                                                              READY   STATUS      AGE
+dagster-daemon-5787ccc868-nsvsg                                   1/1     Running     3m41s
+dagster-dagit-7c5b5c7f5c-rqrf8                                    1/1     Running     3m41s
+dagster-dagster-user-deployments-iris-analysis-564cbcf9f-fbqlw    1/1     Running     3m41s
+dagster-postgresql-0                                              1/1     Running     3m41s
+```
+
+<details>
+  <summary>Debugging failed pods</summary>
+
+If one of the pods is in an error state, you can view the logs using the command
+
+```bash
+kubectl logs <pod-name>
+```
+
+For example, if the pod `dagster-dagster-user-deployments-iris-analysis-564cbcf9f-fbqlw` is in a `CrashLoopBackOff` state, the logs can be viewed with the command
+
+```
+kubectl logs dagster-dagster-user-deployments-iris-analysis-564cbcf9f-fbqlw
+```
+
+</details>
+
+TODO - maybe explain what each of these pods are?
+
+## Step 7: Connect to your Dagster deployment and materialize your assets
+
+### Step 7.1: Start port-forwarding to the webserver pod
+Run the following command to set up port forwarding to the webserver pod"
+
+```bash
+DAGSTER_WEBSERVER_POD_NAME=$(kubectl get pods --namespace default \
+  -l "app.kubernetes.io/name=dagster,app.kubernetes.io/instance=dagster,component=dagster-webserver" \
+  -o jsonpath="{.items[0].metadata.name}")
+kubectl --namespace default port-forward $DAGSTER_WEBSERVER_POD_NAME 8080:80
+```
+
+This command gets the full name of the `webserver` pod from the output of `kubectl get pods`, and then sets up port forwarding with the `kubectl port-forward` command.
+
+### Step 7.2: Visit your Dagster deployment
+The webserver has been port-forwarded to `8080`, so you can visit the Dagster deployment by going to [http://127.0.0.1:8080](http://127.0.0.1:8080). You should see the Dagster landing page
+
+TODO SCREENSHOT
+
+
+### Step 7.3: Materialize an asset
+From the Dagster UI you can materialize an asset by clicking the "Materialize" button. Dagster will start a Kubernetes job to materialize the asset. You can introspect on the Kubernetes cluster to see this job:
```

Comment:
```suggestion
In the Dagster UI, click the **Materialize** button to materialize an asset. Dagster will start a Kubernetes job to materialize the asset. You can introspect on the Kubernetes cluster to see this job:
```

Some light suggestions here, but where in the UI should the user find & click **Materialize**? I don't think it needs to be overly descriptive, but just pointing folks to a page (ex: "The Asset catalog") would be enough